### PR TITLE
made instructions more clear

### PR DESCRIPTION
**Pull request by jacc**
The instructions weren't clear enough so I made them better.

### CODE REVIEW

**Comment by aurieh** [comment]
👀

**Comment by jaacks** [comment]
👀

**Comment by jakeoid** [comment]
```
+5. Run `./script.sh`
+6. Put your token inside of `conf.cfg`, in `userkey="YOURUSERKEY"`
```

Makes no sense to run the script if there isn't a token, perhaps fix this?

**Review by jakeoid — CHANGES_REQUESTED** [review]
Needs to fix points 5. and 6. around.

File: README.md [hunk 1]
```diff
@@ -21,14 +21,15 @@ Thank you to [jomo](https://github.com/jomo/) and his script [jomo/imgur-screens
...
-##instructions
+#Instructions
 
 1. Clone the repo with `git clone https://github.com/whats-this/owo.sh.git`
 2. Make sure `script.sh` has permissions: `chmod a+x script.sh`.
-3. Download the dependencies, you can check what you need with `./script.sh --check`
-4. Put your token inside of `conf.cfg`, in `userkey="YOURUSERKEY"`
-5. Run the command `./setup.sh` to add the owo command.
-6. You can now run `owo file.png` from anywhere to upload `file.png`.
+3. Run `sudo ./setup.sh`
```
Comment (islacantwin) [inline README.md:28]: It shouldn't be getting ran with sudo

**Review by islacantwin — CHANGES_REQUESTED** [review]
What @jakeoid said, and it should say to run anything as sudo.

**Comment by jakeoid** [comment]
No response in over 22 hours, doesn't seem to want to fix the changes on his own.

/closed

**Pull request closed without merging**